runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -340,6 +368,27 @@ public void invalidateClientCache() {
         readCache.invalidateAll();
     }
 
+    /**
+     * Invalidate cache entries of which the keys are less than the specific address.
```

Comment:
nit:Invalidate cache entries with keys less than the specific address.

Reply:
Done.